ludwig/encoders/image_encoders.py

```diff
@@ -127,61 +131,64 @@ def __init__(
             default_weights_regularizer=fc_weights_regularizer,
             default_bias_regularizer=fc_bias_regularizer,
             default_activity_regularizer=fc_activity_regularizer,
-            # default_weights_constraint=fc_weights_constraint,
-            # default_bias_constraint=fc_bias_constraint,
             default_norm=fc_norm,
             default_norm_params=fc_norm_params,
             default_activation=fc_activation,
             default_dropout=fc_dropout,
         )
 
-    def call(self, inputs, training=None, mask=None):
+    def call(self, inputs):
         """
             :param inputs: The inputs fed into the encoder.
-                    Shape: [batch x height x width x channels], type tf.uint8
+                    Shape: [batch x height x width x channels], type torch.uint8
```

Comment:
Has this been changed to NCHW here, or is that still to be.done?

Reply:
This should be done.

Reply:
I need to test everything out, but I've updated image utils to use torchvision.